tests/ert3/config/test_ensemble_config.py

```diff
@@ -121,3 +122,12 @@ def test_unknown_field_in_forward_model(base_ensemble_config):
         ert3.exceptions.ConfigValidationError, match="extra fields not permitted"
     ):
         ert3.config.load_ensemble_config(base_ensemble_config)
+
+
+def test_missing_ouput(base_ensemble_config):
```

Comment:
Have a look at the other input tests above to increase the test coverage on output :)